### ISSUE TITLE
[Screen reader - Make Code - Import] Narrator focus is not synchronized with keyboard focus under Import file section.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2357]
**User Experience:**
Screen reader user might get difficulty in navigation to interact with control if narrator focus doesn’t land on any control as user won't be able to reach out that control. Not only this, they won't be able to get relevant information in respect of that control and also he/she will skip further

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome
Screen reader: Narrator, JAWS, NVDA

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to import file option and select it.
6. Now, navigate to 'Browse' control.
7. Verify keyboard focus and narrator focus on 'Browse' control.

**Actual Result:**
Narrator focus is not synchronized with keyboard focus on 'Browse' control under Import file dialog screen due to this, screen reader user is unable to interact control and also won't be able to take action on such control. Not only this, he/she would miss the functionality and would skip further. 

**Expected Result:**
Narrator focus should synchronized with keyboard focus on 'Browse' control under Import file dialog screen so that screen reader user could easily interact with the control.

**Note:**
1. This issue also exists with Chrome + JAWS.
2. This issue also exists with Chrome + NVDA.

**MAS Reference:**
[MAS 4.2.10 - Tracking of Focus and Selection Attributes (33J)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7bb24201d1-2276-4471-8183-e1e5353031ad%7d)

[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

[MAS4.2.10_Import.zip](https://github.com/microsoft/pxt-microbit/files/3472224/MAS4.2.10_Import.zip)

**Comment by abchatra** [comment]
Please re-test using beta site https://makecode.microbit.org/beta . If it reproduces let me know. I will reopen the bug.
